app/controllers/dashboard_controller.rb

```diff
@@ -97,4 +75,21 @@ def host_warning
                     "It is advised to access velum via one of the registered hosts " \
                     "#{accessible_hosts.join(" or ")}"
   end
+
+  # This method will copy *all* instance variables of the given object to the
+  # current instance. No checks will be performed if a variable already exists -
+  # the current value will be overwritten in that case.
+  #
+  # This copying right now is used to prevent nesting values in the template:
+  # instead of writing `<%= other.var %>` it is possible to just write `<%= var
+  # %>` again. It was solely added as there are tests that expect this syntax,
+  # when rendering a template, and would fail otherwise:
+  # /spec/views/dashboard/autoyast.xml.slim_spec.rb
```

Comment:
the way you do it is correct i guess, and does not only apply to the test, but also to the rendering itself (e.g. when visiting `/autoyast`). so i think i'd remove the bit about the test again